flake.nix

```diff
@@ -83,16 +83,17 @@
               ''
               else "";
 
-            postInstall =
-              if system == "x86_64-linux" then ''
-                chmod +x ./burrito_out/*
-                cp -r ./burrito_out "$out"
+            postInstall = ''
+              chmod +x ./burrito_out/*
+              cp -r ./burrito_out "$out"
 
+              if [ "${system}" == "x86_64-linux" ]; then
                 patchelf --set-interpreter ${pkgs.glibc}/lib/ld-linux-x86-64.so.2 "$out/burrito_out/next_ls_linux_amd64"
-              '' else ''
-                chmod +x ./burrito_out/*
-                cp -r ./burrito_out "$out"
-              '';
+              fi
+              if [ "${system}" == "aarch64-linux" ]; then
```

Comment:
It didn't like when I built the flake on Darwin when you do the branching at the shell level, which is why I broke it out to the nix level